services/ui-src/src/components/pages/ReviewSubmit/McparReviewSubmitPage.tsx

```diff
@@ -96,6 +99,7 @@ export const McparReviewSubmitPage = () => {
 const PrintButton = () => {
   const { print } = reviewVerbiage;
   return (
+    // TODO: make the path route to the correct report type (in the future)
```

Comment:
Do we need to worry about this atm? And if we want to push this down the line is this linked to a JIRA ticket so we don't forget?

Reply:
This is actually no longer a to-do list item, we're going to be creating a separate Review and Submit page for MLR 💯